src/App.jsx

```diff
@@ -14,70 +15,140 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {!selectedGood
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        ) : (
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
+              className="delete ml-3"
+              onClick={() => setSelectedGood('')}
+            />
+          </h1>
+        )
+      }
+
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              className={good === selectedGood
+                ? 'has-background-success-light'
+                : ''
+              }
             >
-              +
-            </button>
-          </td>
+              <td>
+                {good !== selectedGood
+                  ? (
+                    <button
+                      data-cy="AddButton"
+                      type="button"
+                      className="button"
+                      onClick={() => setSelectedGood(good)}
+                    >
```

Comment:
You use `button`  in both cases. Consider moving the ternary operator inside the tag

Reply:
Thank you for your suggestion, but I'm considering using the ternary operator for three of four button attributes less readable than just leaving the whole button itself.